From ee99074adb5eb1c05783ec06695023320a35cb70 Mon Sep 17 00:00:00 2001
From: Praveen Diwakar <praveen.diwakar@intel.com>
Date: Fri, 14 Nov 2014 04:42:23 +0530
Subject: [PATCH 5/7] ASoC: Intel: sst: Remove recovery code for hostless
 stream

GMIN does not currently have voice call feature, which is
hostless based stream. Hence given code can be removed.

Issue: GMINL-3666

Change-Id: I7f1eb295061b09dfb50ddc6ca20057266de6216b
Signed-off-by: Praveen Diwakar <praveen.diwakar@intel.com>
---
 sound/soc/intel/sst/sst_pvt.c |    9 ---------
 1 file changed, 9 deletions(-)

diff --git a/sound/soc/intel/sst/sst_pvt.c b/sound/soc/intel/sst/sst_pvt.c
index bdfb5f6..3db9bad 100644
--- a/sound/soc/intel/sst/sst_pvt.c
+++ b/sound/soc/intel/sst/sst_pvt.c
@@ -360,8 +360,6 @@ void sst_do_recovery_mrfld(struct intel_sst_drv *sst)
 	char iram_event[30], dram_event[30], ddr_imr_event[65], event_type[30];
 	char *envp[5];
 	int env_offset = 0;
-	bool reset_dapm;
-	struct sst_platform_cb_params cb_params;
 
 	/*
 	 * setting firmware state as RESET so that the firmware will get
@@ -375,11 +373,6 @@ void sst_do_recovery_mrfld(struct intel_sst_drv *sst)
 	sst->sst_state = SST_RECOVERY;
 	mutex_unlock(&sst->sst_lock);
 
-	cb_params.params = &reset_dapm;
-	cb_params.event = SST_PLATFORM_TRIGGER_RECOVERY;
-	reset_dapm = true;
-	sst_platform_cb(&cb_params);
-
 	sst_stall_lpe_n_wait(sst);
 
 	/* dump mailbox and sram */
@@ -444,8 +437,6 @@ void sst_do_recovery_mrfld(struct intel_sst_drv *sst)
 	 * powering on DAPM widget
 	 */
 	usleep_range(10000, 12000);
-	reset_dapm = false;
-	sst_platform_cb(&cb_params);
 }
 
 void sst_do_recovery(struct intel_sst_drv *sst)
-- 
1.7.9.5

